luminoso_api/v5_client.py

```diff
@@ -152,9 +152,14 @@ def save_token(cls, token=None, domain='daylight.luminoso.com',
             domain = domain.split('/')[0]
 
         if token is None:
-            default_url = f'https://{domain}/'
-            url = input(f'URL (or press enter if "{default_url}" is'
-                        f' correct): ') or default_url
+            protocol = None
+            while protocol is None:
+                prompt = input('Use https? (y/n, default=y): ').lower()
```

Comment:
Okay, so, this makes sense, but if we're doing this, I think:
- If they pass `domain` as something like `https://daylight.luminoso.com` or `http://daylight-master` in the first place, then we should use the protocol they gave, rather than prompting for one
- `domain` should default to `https://daylight.luminoso.com`

Reply:
Went back and forth on the latter several times, and decided not to change the default for `domain`, because tokens are saved per *domain*, not per *URL* (i.e., if you somehow tried to save one for `http://daylight.luminoso.com/` and then one for `https://daylight.luminoso.com/`, the latter would overwrite the former), so I don't really want the protocol as part of the default domain argument.  But the function does check for `daylight.luminoso.com` and sets the protocol in that case to `https`.

Reply:
👍